test/test-build.html

```diff
@@ -17,14 +17,17 @@
 
     System.config({
       baseURL: './fixtures/test-tree/',
+      meta: {
+         'output/*': { format: 'register' }
+      },
```

Comment:
I think for this reason we may need to make this change a 0.16 builder PR, instead of adding this to 0.15.